### ISSUE TITLE
ImportError: libkea.so.1.4.7: cannot open shared object file: No such file or directory

### ISSUE THREAD

**Issue by hugovk** [darribas/contextily#42]
Latest master passed on Python 3 four days ago:

https://travis-ci.org/darribas/contextily/builds/407264253

It now fails on Python 3:

```
==================================== ERRORS ====================================
______________________ ERROR collecting tests/test_ctx.py ______________________
ImportError while importing test module '/home/travis/build/hugovk/contextily/tests/test_ctx.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
tests/test_ctx.py:3: in <module>
    import contextily as ctx
contextily/__init__.py:6: in <module>
    from .place import Place, plot_map
contextily/place.py:6: in <module>
    from .tile import howmany, bounds2raster, bounds2img, _sm2ll, _calculate_zoom
contextily/tile.py:11: in <module>
    import rasterio as rio
miniconda/envs/TEST/lib/python3.5/site-packages/rasterio/__init__.py:23: in <module>
    from rasterio._base import gdal_version
E   ImportError: libkea.so.1.4.7: cannot open shared object file: No such file or directory
___________________ ERROR collecting tests/test_providers.py ___________________
ImportError while importing test module '/home/travis/build/hugovk/contextily/tests/test_providers.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
tests/test_providers.py:1: in <module>
    import contextily as ctx
contextily/__init__.py:6: in <module>
    from .place import Place, plot_map
contextily/place.py:6: in <module>
    from .tile import howmany, bounds2raster, bounds2img, _sm2ll, _calculate_zoom
contextily/tile.py:11: in <module>
    import rasterio as rio
miniconda/envs/TEST/lib/python3.5/site-packages/rasterio/__init__.py:23: in <module>
    from rasterio._base import gdal_version
E   ImportError: libkea.so.1.4.7: cannot open shared object file: No such file or directory
```

https://travis-ci.org/hugovk/contextily/builds/408951523

This also affects PRs, such as #40. See also some extra info at https://github.com/darribas/contextily/pull/40#issuecomment-408434736.

Could this be a problem with a Conda update, similar to https://github.com/conda-forge/fiona-feedstock/issues/86?

**Comment by ljwolf** [comment]
Note that [libgdal is not being installed from the forge](https://travis-ci.org/darribas/contextily/jobs/407480023#L588)? try explicitly installing it from the forge?

**Comment by hugovk** [comment]
Both passing and failing builds both use libgdal-2.2.4.

I'm not familiar with Conda, how should it be explicitly installed from the forge?

**Comment by darribas** [comment]
This seems to be resolved as #43 passed correctly without any changes on the infrastructure side... Closing for now but, if resurfaces, we can reopen.
